man/manual.md

```diff
@@ -312,13 +334,20 @@ Another useful flag might be `-U`. It tells Cppcheck that a macro is not defined
 
 To add an include path, use `-I`, followed by the path.
 
-Cppcheck's preprocessor basically handles includes like any other preprocessor. However, while other preprocessors stop working when they encounter a missing header, cppcheck will just print an information message and continues parsing the code.
+Cppcheck's preprocessor basically handles includes like any other preprocessor. 
+However, while other preprocessors stop working when they encounter a missing header, Cppcheck will just print an 
+information message and continues parsing the code.
 
-The purpose of this behaviour is that cppcheck is meant to work without necessarily seeing the entire code. Actually, it is recommended to not give all include paths. While it is useful for cppcheck to see the declaration of a class when checking the implementation of its members, passing standard library headers is highly discouraged because it will result in worse results and longer checking time. For such cases, .cfg files (see below) are the better way to provide information about the implementation of functions and types to cppcheck.
+The purpose of this behaviour is that Cppcheck is meant to work without necessarily seeing the entire code. 
+Actually, it is recommended to not give all include paths. 
+While it is useful for Cppcheck to see the declaration of a class when checking the implementation of its members, 
+passing standard library headers is discouraged, because it will result in less informative results and longer checking 
```

Comment:
I do not think "less informative" is true. I believe that the analysis will not work fully.